applications/ShapeOptimizationApplication/custom_utilities/mapping/filter_function.h

```diff
@@ -94,9 +95,13 @@ class FilterFunction
         else if (filter_function_type.compare(linear) == 0)
             m_filter_function_type = 2;
 
+        // Type 3: Constant function
+        else if (filter_function_type.compare(constant) == 0)
+            m_filter_function_type = 2;
+
         // Throw error message in case of wrong specification
         else
-            KRATOS_ERROR << "Specified filter function type not recognized. Options are: linear , gaussian. Specified: " << std::endl;
+            KRATOS_ERROR << "Specified filter function type not recognized. Options are: constant, linear , gaussian. Specified: " << std::endl;

```

Comment:
this is quite manual

alternative see [here](https://github.com/KratosMultiphysics/Kratos/blob/meshmoving/claculate-mesh-vel-utility/applications/MeshMovingApplication/custom_utilities/calculate_mesh_velocity_utility.cpp#L45-L73)